File: MovieTicketBooking/Movie Booking Java/src/main/java/org/example/model/Show.java
package org.example.model;

import lombok.AllArgsConstructor;
import lombok.Getter;

import java.time.LocalDateTime;

@AllArgsConstructor
@Getter
public class Show {

    private final String id;
    private final Screen screen;
    private final LocalDateTime startTime;
    private final Integer durationInMinutes;
    private final Movie movie;
}

File: MovieTicketBooking/Movie Booking C#/Model/Show.cs
using System.Data.Common;

namespace MovieTicketBooking.Model
{
    public class Show
    {
        string Id;
        Movie Movie;
        int DurationInMin;
        DateTime StartTime;
        public Screen Screen { get; }
        public Show(string id, Movie movie, int durationInMins, DateTime startTime, Screen screen)
        {
            this.Id = id;
            this.Movie = movie;
            this.DurationInMin = durationInMins;
            this.StartTime = startTime;
            this.Screen = screen;
        }
    }
}